src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<ul><li>"constant(or profile or zero-lift)" --&gt; "constant (or profile or zero-lift)"</li><li>"&nbsp;are constant" --&gt; "&nbsp;are constants"</li><li>"thourough" --&gt; "thorough"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Corrected in a6b9f26f